tests/test_coverage.py

```diff
@@ -331,6 +333,44 @@ def test_items_that_need_coverage_respects_count_as_covered(self):
             ).all()
         )
 
+    def test_isbn_covers_are_imported_from_mapped_identifiers(self):
+        # Now that we pass ISBN equivalents instead of Bibliotheca identifiers
+        # to the Metadata Wrangler, they're not getting covers. Let's confirm
+        # that the problem isn't on the Circulation Manager import side of things.
+
+        # Create a Bibliotheca identifier with a license pool.
+        source = DataSource.lookup(self._db, DataSource.BIBLIOTHECA)
+        identifier = self._identifier(identifier_type=Identifier.BIBLIOTHECA_ID)
+        LicensePool.for_foreign_id(
+            self._db, source, identifier.type, identifier.identifier
+        )
+
+        # Create an ISBN and set it equivalent.
+        isbn = self._identifier(identifier_type=Identifier.ISBN)
+        isbn.identifier = '9781594632556'
+        identifier.equivalent_to(source, isbn, 1)
+
+        opds = sample_data('metadata_isbn_response.opds', 'opds')
+        self.provider.lookup.queue_response(
+            200, {'content-type': 'application/atom+xml;profile=opds-catalog;kind=acquisition'}, opds
+        )
+
+        result = self.provider.process_item(identifier)
+        # The lookup is successful
+        eq_(result, identifier)
+        # The appropriate cover links are transferred.
+        identifier_uris = [l.resource.url for l in identifier.links
+                           if l.rel in [Hyperlink.IMAGE, Hyperlink.THUMBNAIL_IMAGE]]
+        expected = [
+            'http://book-covers.nypl.org/Content%20Cafe/ISBN/9781594632556/cover.jpg',
+            'http://book-covers.nypl.org/scaled/300/Content%20Cafe/ISBN/9781594632556/cover.jpg'
+        ]
+
+        eq_(sorted(identifier_uris), sorted(expected))
+
+        # The ISBN doesn't get any information.
+        eq_(isbn.links, [])
```

Comment:
Argument for: putting this information on the ISBN would be more accurate. It would let us get rid of the reverse mapping when data comes back from the metadata wrangler.

Counterargument: if core metadata comes back from the metadata wrangler (like a correction to the book's title or contributors), associating it with the ISBN won't be useful because we only find data associated with the ISBN when it's associated with the Identifier. That means classifications, descriptions, and cover images, but not title or contributors, which are associated with the Edition.

I think leave it alone for now.